doc/languages-frameworks/go.xml

```diff
@@ -221,4 +221,34 @@ done
 </screen>
   </para>
  </section>
+ <section xml:id="ssec-go-common-attributes">
+  <title>Attributes used by the builders</title>
+   <para>Both <link xlink:href="#ssec-go-modules"><varname>buildGoModule</varname></link> and <link xlink:href="#ssec-go-modules"><varname>buildGoPackage</varname></link> Read the attributes <varname>buildFlagsArray</varname> and <varname>buildFlags</varname> to set build flags supported by <varname>go</varname>'s commands such as <varname>go install</varname> and <varname>go build</varname>. Usually, you'd want to use one of the attributes to make the resulting executable aware of it's own version, or alike. For example:</para>
```

Comment:
Maybe remove `buildFlags` and just have `buildFlagsArray`?

Reply:
I think it'd be a lie to not mention `buildFlags` at all, unless we also remove it from the builder. However, we can recommend in the docs to use the array.